### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by matt-goldman**
Fixes: #394 
Fixes: #136 
Fixes: #326 
Fixes: #395
Fixes: #392 
image
image

### CODE REVIEW

File: Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ProfileViewModel.cs [hunk 1]
```diff
@@ -344,7 +344,7 @@ private async Task AddSocialMediaId(SocialUsernameMessage message)
 
             if (result)
             {
-                MessagingCenter.Send(this, Constants.PointsAwardedMessage);
+                MessagingCenter.Send<object>(this, Constants.PointsAwardedMessage);
```
Comment (lukecookssw) [inline Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ProfileViewModel.cs:347]: Is there something a little more explicit that you can type this as? `object` isn't very helpful to anyone.

File: Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ScanResultViewModel.cs [hunk 2]
```diff
@@ -117,7 +117,7 @@ public async Task CheckScanData()
             if (result.result == ScanResult.Added)
             {
                 await _userService.UpdateMyDetailsAsync();
-                MessagingCenter.Send(this, Constants.PointsAwardedMessage);
+                MessagingCenter.Send<object>(this, Constants.PointsAwardedMessage);
```
Comment (lukecookssw) [inline Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ScanResultViewModel.cs:120]: Same here.

**Review by lukecookssw — CHANGES_REQUESTED** [review]

File: Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ProfileViewModel.cs (hunk 1 above)
Comment (matt-goldman) [reply Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ProfileViewModel.cs]: Reverted this change

File: Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ScanResultViewModel.cs (hunk 2 above)
Comment (matt-goldman) [reply Xamarin/SSW.Rewards/SSW.Rewards/ViewModels/ScanResultViewModel.cs]: Reverted this change

**Review by lukecookssw — APPROVED** [review]
not quite what I had in mind but it's still somehow better I guess?

**Pull request merged**